src/tools/kata-ctl/src/utils.rs

```diff
@@ -21,6 +24,22 @@ pub fn drop_privs() -> Result<()> {
     Ok(())
 }
 
+const PROC_VERSION_FILE: &str = "/proc/version";
```

Comment:
I prefer to have the constants defined to where they are used to avoid scrolling hundreds of lines as the code grows. But I am not sure if the project follows the practice of having the constants at the top of the file for rust code honestly.